docs/modules/ROOT/pages/type-definitions/schema-configuration/index.adoc

```diff
@@ -0,0 +1,37 @@
+[[type-definitions-schema-configuration]]
+= Schema Configuration
+
+Neo4j GraphQL Library supports several functionalities like CRUD operations, aggregation, filtering, and many others.
+
+To make them work, a large amount of GraphQL types are generated.
+
+For instance, starting from the type definitions:
+
+[source, graphql, indent=0]
+----
+type Movie { 
+    title: String
+    length: Int
+} 
+----
+
+
+In some cases, it may be advisable to reduce the scope of the API produced. Whatever the reason is to reduce the size of the schema, improve the performance, or limit access to unwanted operations, this section provides all the information needed to configure the API produced.
```

Comment:
This is somewhat duplicate content from type-definitions/schema-configuration.adoc but there are a few differences. For instance, there it says "in most cases" instead of "some cases". Also, it's advisable not to have the same content in different pages as this can make Google indexing worse.